e2e/pod.go

```
@@ -105,17 +105,49 @@ func waitForDeploymentComplete(name, ns string, c kubernetes.Interface, t int) e
 	return nil
 }
 
-func getCommandInPodOpts(f *framework.Framework, c, ns string, opt *metav1.ListOptions) (framework.ExecOptions, error) {
+func findContainerName(f *framework.Framework, c, ns, cn string, opt *metav1.ListOptions) (string, string, error) {
+	pn := ""
+	podList, err := f.PodClientNS(ns).List(context.TODO(), *opt)
+	framework.ExpectNoError(err)
```

return error?

---

This is an assertion, we are returning at line 113

---

what I meant was as we are returning an error from this function, let's return an error from here and the caller handles it. let's not do both assertion and error returning from the same functions

---

@pkalever any comment on this one?

---

Ah! Got you, I will remove this line, thanks for the clarification.